tools/cpp/windows_cc_configure.bzl

```diff
@@ -297,7 +299,22 @@ def _is_support_winsdk_selection(repository_ctx, vc_path):
         if vc_common_ide.get_child(tool).exists:
             return True
     return False
+    
+def _get_vc_env_vars(repository_ctx, vc_path, msvc_vars_x64, target_arch):
```

Comment:
This is a nice solution, can you put a document here to explain what does this function do and why do we need it?